lib/ruby-debug-ide/command.rb

```diff
@@ -120,6 +120,11 @@ def debug_eval(str, b = get_binding)
       end
     end
 
+    def unescape_incoming(str)
```

Comment:
imho it would be nice to provide a unit test for the method (in terms "string in - string out)